src/container/list/list_spec.gobra

```diff
@@ -10,6 +10,7 @@ requires list.Mem(elems, true)
 requires e in elems
 ensures  res == (unfolding list.Mem(elems, true) in e.next)
 ensures  unfolding list.Mem(elems, true) in res in elems
+decreases _

```

Comment:
Can you please resolve the issues that you have addressed and that are not pending on answers to your questions? This would make it easy for me to understand what is still open